data/projects/editor_support.md

```diff
@@ -1,31 +1,68 @@
-Develop an efficient strategy that allows coala to maintain a number of plugins
-for different editors.
+coala features quite a few editor plugins already, none of which are 
+*supported* by the definition of having regular continuous integration and 
+tests running to ensure its functionality. The goal of this project is to 
+plan, introduce and configure a testing routine that allows coala to ensure
```

Comment:
Maybe add the problems we have due to the lack of testing aka plugins not working and nobody notices it.